src/styles/main.scss

```diff
@@ -1,3 +1,263 @@
-body {
-  margin: 0;
+@import "../styles/utils/variables";
+@import "../styles/utils/mixin";
+@import "../styles/stars-block/stars";
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
```

Comment:
each BEM block should be in a separate file (according to BEM rules)

Reply:
![Screenshot_6](https://github.com/mate-academy/layout_catalog/assets/149514552/ffca1196-b7c2-4520-8178-ffc68ee125be)